s96at/src/i2c_linux.c

```diff
@@ -64,12 +64,39 @@ static uint32_t i2c_linux_close(void *ctx)
 	return close(ictx->fd) == 0 ? STATUS_OK : STATUS_EXEC_ERROR;
 }
 
+/* As Linux ioctls do not provide a way to control the I2C lines directly,
+ * we send a request to address 0x00, which pulls SDA low for 7 cycles.
+ * This method is not failsafe as the wakeup low duration is 60 usec
+ * and the device can operate on frequencies up to 1MHz. This will therefore
+ * only work on systems clocked up to 133KHz.
```

Comment:
It would be nice to give build warning or something like that when you are trying open the device with frequencies higher than 133KHz, but I don't think there is an easy way to do that. I guess that would be both device dependant (platform data) as well as in some cases they are just using Device Tree. Anyhow, the comment is a very good idea for now, thanks!